README.md

```diff
@@ -32,7 +32,7 @@ installed with a variety of plugin managers:
 <leader><space> : Preview the quickfix/location result in a preview window
 <enter>         : Open the quickfix/location result in a new buffer like usual
 ```
-P.S.: `\` is the leader key by default. So the mapping is `\<space>` unless `<leader>`
+P.S.: `\\` is the leader key by default. So the mapping is `\<space>` unless `<leader>`
```

Comment:
The vim syntax plugin I am using for markdown did not like

    '\'

but it was happy with

    '\\'

and that also looks fine on Github's web view. It would appear that is more correct. (See the rightmost column of this [Github PDF doc](https://enterprise.github.com/downloads/en/markdown-cheatsheet.pdf).)